Dockerfile.ubuntu16.04.bats

```diff
@@ -3,17 +3,16 @@ MAINTAINER Gruntwork <info@gruntwork.io>
 
 # Install basic dependencies
 RUN DEBIAN_FRONTEND=noninteractive apt-get update && \
-    apt-get install -y vim python-pip jq sudo curl libffi-dev python3-dev && \
+    apt-get install -y vim git python-pip jq sudo curl libffi-dev python3-dev && \
     update-alternatives --install /usr/bin/python python /usr/bin/python3 1 && \
     update-alternatives --config python && \
     curl https://bootstrap.pypa.io/pip/3.5/get-pip.py -o /tmp/get-pip.py && \
     python /tmp/get-pip.py
 
 # Install Bats
-RUN apt-get install -y software-properties-common && \
-    add-apt-repository ppa:duggan/bats && \
-    DEBIAN_FRONTEND=noninteractive apt-get update && \
-    apt-get install -y bats
+RUN git clone https://github.com/bats-core/bats-core.git /tmp/bats-core && \
+    /tmp/bats-core/install.sh /usr/local && \
+    rm -r /tmp/bats-core
```

Comment:
I couldn't find good documentation on the `duggan/bats` PPA, and it turns out that isn't supported on Ubuntu 20.04, so I switched to [the officially recommended way to install bats](https://github.com/bats-core/bats-core).